app/components/QuakeCard.tsx

```diff
@@ -49,44 +51,63 @@ const QuakeCard = ({ place, code, coords, mag, time, url }: QuakeCardProps) => {
   };
 
   return (
-    <div
-      className="relative bg-gray-100 rounded-lg shadow-lg cursor-pointer p-3 text-color-purple-heart-950"
-      onClick={() => onZoom(coords)}
-    >
-      {/* <button
-        className="absolute top-[10px] right-[10px]"
-        onClick={(e) => {
-          removeQuake();
-          e.stopPropagation();
-        }}
-      >
-        <AiOutlineClose color="#5236cc" />
-      </button> */}
+    <div className="relative bg-orange-50 rounded-lg shadow-lg py-4 px-6 text-color-purple-heart-950">
       <div
-        className={classNames(
-          'flex place-content-center p-[2px] mx-[16px] shadow-sm rounded-lg',
-          magClass
-        )}
+        style={{
+          display: 'flex',
+          gap: '16px',
+          justifyContent: 'space-between',
+        }}
       >
-        <p
-          style={{
-            fontSize: '22px',
-            fontWeight: 'bold',
-            color: 'white',
+        <div
+          className={classNames(
+            'px-2 py-1 w-min shadow-sm rounded-lg',
+            magClass
+          )}
+        >
+          <p
+            style={{
```

Comment:
Make sure these get converted to Tailwind classes... but on that note -

These class names seem like they can get out of hand really fast - are there commonly used tools to manage them?